src/views/Run/internal/ResultDisplay/internal/ReportSection/ReportSection.tsx

```diff
@@ -0,0 +1,54 @@
+import { FC } from 'react'
+
+import { RunReport } from '@/benchttp'
+import { isDurationMetricField } from '@/benchttp/metrics'
+import { Tag, Typography } from '@/components'
+import { convertNsToMs } from '@/tools/converters'
+import { parseMilliseconds } from '@/tools/parsers'
+
+import { TestResult } from './internal/components'
+import { getFailedTestCount } from './internal/ReportSection.helpers'
+
+export interface IProps {
+  report: RunReport
+}
+
+export const ReportSection: FC<IProps> = ({ report }) => {
+  const failedTestCount: number = getFailedTestCount(report.tests.results)
+
+  return (
+    <div>
+      <div className="f f-ai-center mb-4">
+        <Typography className="mr-1" element="h1">
+          Status :
```

Comment:
Still a nitpick, but I actually meant:
```suggestion
          Status: 
```
:sweat_smile: